### ISSUE TITLE
API-10:  JDBC 4.3

### ISSUE THREAD

**Issue by njr-11** [OpenLiberty/open-liberty#4728]
jdbc-4.3 feature and full integration with JDBC 4.3 spec for Liberty

**Comment by atosak** [comment]
Ship readiness checklist is being tracked here [](https://github.com/OpenLiberty/open-liberty/issues/5431)

**Comment by marikaj123** [comment]
Translation is required, waiting for shipment 4 to arrive.

**Comment by gscottj** [comment]
no user interface

**Comment by donbourne** [comment]
@hanczaryk , when you're done with SVT on this one, can you please indicate if you're happy that the serviceability of the problems you encountered is sufficient to avoid PMRs, or that L2 should be able to quickly address those problems without need to engage L3?

**Comment by hanczaryk** [comment]
@donbourne Yes, the serviceability of the problems encountered were sufficient to avoid PMRs or L2 should be able to quickly address.

**Comment by donbourne** [comment]
thanks @hanczaryk

**Comment by marikaj123** [comment]
Nathan confirmed the feature is in a green Liberty release build.
